src/Aspire.Hosting.NodeJs/NodeInstallerResource.cs

```diff
@@ -6,9 +6,9 @@
 namespace Aspire.Hosting.NodeJs;
 
 /// <summary>
-/// A resource that represents an npm package installer.
+/// A resource that represents a package installer for a node app.
 /// </summary>
 /// <param name="name">The name of the resource.</param>
 /// <param name="workingDirectory">The working directory to use for the command.</param>
-public class NpmInstallerResource(string name, string workingDirectory)
-    : ExecutableResource(name, "npm", workingDirectory);
\ No newline at end of file
+public class NodeInstallerResource(string name, string workingDirectory)
```

Comment:
Why wouldn't this just wrap `NodeAppResource`?

Reply:
I'm not sure I understand the question.

There are 2 separate "executable" resources. One that does `npm install` (that's this one), and then one that does `npm run dev`.

This isn't a "NodeApp" - this is the executable that that runs `npm install` for the node app.

Reply:
Sorry, I forgot that the `NodeAppResource` doesn't actually call the `node` executable, it calls whatever you pass to it (If you wanted to be silly you could do `new NodeAppResource("silly", "python", "../app");` since it just passes the executable through.

So using this, if the executable is `node`, won't it try to run `node npm install`? That won't work will it? Or does the `WithCommand` override the executable?

Reply:
> Or does the WithCommand override the executable?

Yep! This exactly. We do the same with the ViteApp - by default it references `node` as the executable. But then the PackageManager calls `WithCommand` to change it to `npm` or `yarn` or whatever.

https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9.5#withcommand-and-withworkingdirectory-apis